portal/index.html

```diff
@@ -0,0 +1,93 @@
+<!doctype html>
```

Comment:
somehow monorepos standard for things like this are more like this, so all webpack crap can be removed from root dir and be isolated inside the directory. Should we do something like this too? 
![image](https://user-images.githubusercontent.com/3225744/111980279-f8c5be80-8afd-11eb-93e1-af0cad8dadd0.png)